AXSwift/Observer.swift

```diff
@@ -104,32 +104,36 @@ public final class Observer {
   ///         error is not passed on for consistency with `start()` and `stop()`.
   /// - throws: `Error.NotificationUnsupported`: The element does not support notifications (note
   ///           that the system-wide element does not support notifications).
-  public func removeNotification(notification: Notification, forElement element: UIElement) throws {
-    let error = AXObserverRemoveNotification(axObserver, element.element, notification.rawValue)
-    guard error == .Success || error == .NotificationNotRegistered else {
+  public func removeNotification(_ notification: AXNotification, forElement element: UIElement) throws {
+    let error = AXObserverRemoveNotification(axObserver, element.element, notification.rawValue as CFString)
+    guard error == .success || error == .notificationNotRegistered else {
       throw error
     }
   }
 }
 
-private func internalCallback(axObserver: AXObserver,
+private func internalCallback(_ axObserver: AXObserver,
                               axElement: AXUIElement,
                               notification: CFString,
-                              userData: UnsafeMutablePointer<Void>) {
-  let observer = Unmanaged<Observer>.fromOpaque(COpaquePointer(userData)).takeUnretainedValue()
-  let element  = UIElement(axElement)
-  let notif    = Notification(rawValue: notification as String)!
-  observer.callback!(observer: observer, element: element, notification: notif)
+                              userData: UnsafeMutableRawPointer?) {
+  if let userData = userData {
```

Comment:
I can't think of any reason why this should be nil, can you? It looks like the old code assumed it was present (and nil would indicate a bug somewhere). I would rather not mask potential bugs if that's still the case.